scraper/src/merge_datasets.js

```diff
@@ -291,6 +422,22 @@ async function mergeDatasets() {
   return allPlants;
 }
 
+/**
+ * Cleans up a JSON array for smoother CSV export.
```

Comment:
```suggestion
 * Removes empty JSON array entries for smoother CSV export.
```

Reply:
This can/should be extended in the future if we want to apply something to all properties of the plants array

Reply:
I don't really understand your [comment](https://github.com/ElektraInitiative/PermaplanT/pull/799#discussion_r1291966768). Code comments should describe what is, not what might be.

@badnames' comment is much clearer to what was there before (cleaning up an array could be nearly anything). Is @badnames' suggestion not correct? If it is, I would prefer the more clear statement what the code does.

Reply:
I extended the function to incorporate the requested changes.